plugins/router/bbox

```diff
@@ -0,0 +1,140 @@
+#! /bin/sh
+# Parse bbox information
```

Comment:
This is a very, very short documentation header :)

Please excuse my ignorance, but I have no idea, what bbox means. Currently I think, it is some kind of network statistics collector. But initially I thought about spatial bounding boxes :)

Maybe you want to add a perldoc header?